### DISCUSSION TITLE
How to have a dependency 2 version behinds and always pick up the 2 version behind

### DISCUSSION THREAD

**Discussion by juancarlosjr97** [renovatebot/renovate#29439] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

I am trying to follow a dependency release but always 2 versions behind, so for example if there is a dependency v10, I want to be v8 and if v11 is released, go to v9.

Is this even possible?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by juancarlosjr97** [comment]
Anyone? :)

**Comment by rarkins** [comment, marked as answer]
Not possible
